docusaurus.config.js

```diff
@@ -186,7 +186,7 @@ module.exports = {
           items: [
             {
               label: "Join the Cloud Waitlist",
```

Comment:
```suggestion
              label: "Join the Cloud waitlist",
```
This keeps catching my eye.